files/es/learn/html/tables/basics/index.md

```diff
@@ -0,0 +1,557 @@
+    </tr>
+    <tr>
+      <th>Hipopótamo</th>
+    </tr>
+    <tr>
+      <th>Caballo</th>
+      <td>Yegua</td>
+    </tr>
+    <tr>
+      <td>Semental</td>
+    </tr>
+    <tr>
+      <th>Cocodrilo</th>
+    </tr>
+    <tr>
+      <th>Pollo</th>
+      <td>Gallina</td>
+    </tr>
+    <tr>
+      <td>Gallo</td>
+    </tr>
+  </tbody>
+</table>
+
+Necesitamos una forma de hacer que «Animales», «Hipopótamo» y «Cocodrilo» se extiendan dos columnas más allá, y «Caballo» y «Pollo» se extiendan dos filas más abajo. Por fortuna, los encabezados de tabla y las celdas tienen los atributos `colspan` y `rowspan`, que nos permiten hacer exactamente esas cosas. Ambos aceptan un valor numérico sin unidades, que es igual al número de filas o columnas que desea abarcar. Por ejemplo, `colspan="2"` extiende una celda dos columnas más allá.
+
+Usemos `colspan` y `rowspan` para mejorar esta tabla.
+
+1.  Primero, haz una copia local de nuestros archivos [animals-table.html](https://github.com/mdn/learning-area/blob/master/html/tables/basic/animals-table.html) y [minimal-table.css](https://github.com/mdn/learning-area/blob/master/html/tables/basic/minimal-table.css) en un directorio nuevo de tu ordenador. El HTML contiene el mismo ejemplo sobre perros que viste arriba.
+2.  Luego, usa `colspan` para extender las celdas «Animales», «Hipopótamo» y «Cocodrilo» dos columnas más allá.
+3.  Por último, usa `rowspan` para extender las celdas de «Caballo» y «Pollo» dos filas más abajo.
+4.  Guarda tu código y ábrelo en un navegador para ver la mejora.
+
+> **Nota:** Puedes encontrar nuestro ejemplo terminado en [animals-table-fixed.html](https://github.com/mdn/learning-area/blob/master/html/tables/basic/animals-table-fixed.html) en GitHub ([o consultarlo en vivo](http://mdn.github.io/learning-area/html/tables/basic/animals-table-fixed.html)).
+
+## Proporcionar un estilo común a las columnas.
+
+Hay una última característica de la que queremos hablar en este artículo antes de continuar. El HTML tiene un método para definir información de estilo para una columna completa de datos en un solo lugar: los elementos **[`<col>`](/es/docs/Web/HTML/Elemento/col)** y **[`<colgroup>`](/es/docs/Web/HTML/Elemento/colgroup)**. Estos atributos existen porque especificar el estilo de las columnas puede resultar enojoso e ineficiente; en general hay que especificar la información de estilo en _cada_ `<td>` o `<th>` de la columna, o utilizar un selector complejo como {{cssxref(":nth-child()")}}.
+
+Tomemos el ejemplo sencillo siguiente:
+
+```html
+<table>
+  <tr>
+
+<th>Dato 1</th>
+    <th style="background-color: yellow">Dato 2</th>
+  </tr>
+  <tr>
+    <td>Calcuta</td>
+    <td style="background-color: yellow">Pizza</td>
+  </tr>
+  <tr>
+    <td>Robots</td>
+    <td style="background-color: yellow">Jazz</td>
+  </tr>
+</table>
+```
+
+Esto nos da el resultado siguiente:
+
+| Dato 1  | Dato 2  |
+| ------- | ------- |
+| Calcuta | Naranja |
+| Robots  | Jazz    |
```

Comment:
Fix format and example

````suggestion
```html
<table>
  <tr>
    <th>Dato 1</th>
    <th style="background-color: yellow">Dato 2</th>
  </tr>
  <tr>
    <td>Calcuta</td>
    <td style="background-color: yellow">Pizza</td>
  </tr>
  <tr>
    <td>Robots</td>
    <td style="background-color: yellow">Jazz</td>
  </tr>
</table>
```

Esto nos da el resultado siguiente:

<table>
  <tr>
    <th>Dato 1</th>
    <th style="background-color: yellow">Dato 2</th>
  </tr>
  <tr>
    <td>Calcuta</td>
    <td style="background-color: yellow">Pizza</td>
  </tr>
  <tr>
    <td>Robots</td>
    <td style="background-color: yellow">Jazz</td>
  </tr>
</table>
````